apps/fishing-map/features/map/popups/TileClusterLayers.tsx

```diff
@@ -94,7 +94,7 @@ function EncounterTooltipRow({ feature, showFeaturesDetails }: EncountersLayerPr
 
   const event = parseEvent(feature.event)
   const linkParams = {
-    ...viewport,
+    ...viewState,
```

Comment:
This is going to include a lot of params that we don't need (like maxPitch, minPitch,minZoom,normalize,pitch...) when we only need latitude, longitude, and zoom.
What do you think of having a selector in the atom to return just these three properties?